vite-project/src/components/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { IcArrow, IcHome } from '../assets/svgs/icons';
+
+const Header = () => {
+    return (
+        <Head>
+            <IcArrow />
+            <HeadTextWrapper>
+                장바구니 <HeadTextBox>(2)</HeadTextBox>
+            </HeadTextWrapper>
+            <IcHome />
+        </Head>
+    );
+};
+
+export default Header;
+
+const Head = styled.div`
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding: 0 1.2rem 1.2rem 1.2rem;
```

Comment:
```suggestion
    padding: 0 1.2rem 1.2rem;
```
이렇게 작성하면 순서대로 `상` `좌우` `하` 이렇게 된다! ㅎㅎ

Reply:
오호.. 위 오른쪽 아래 왼쪽 이라고만 기억했는데 알겠습니당!